### ISSUE TITLE
Attributes are locked on registration

### ISSUE THREAD

**Issue by iambodo** [HRHR-project/e-registry-app#90]
When enrolling a new client, all attributes are locked/blacked out, as if they have been pre-filled. No errors in client side logs. 

However, after refreshing the page, this error disappears and can enroll new client like normal. As we are in training, this presents a major workflow bottleneck for new users.

Model control user
afreen_TEST
eReg!234
![image](https://user-images.githubusercontent.com/36451485/39701593-0f2fbff0-5202-11e8-9b13-a7415ec42ced.png)

**Comment by AkubaD** [comment]
Hi @lukasewvd  @Bekkalizer 
Could you take a look at this error? I just experienced it today with my admin user and no particularly unique workflow. 
Looks like it is one of the intermittent errors we have been seeing lately which can be difficult to replicate. Please let us know what more information you need to be able to investigate.
As Brian mentioned in the initial issue, it goes away once you refresh the page

**Comment by RahmanMonjur** [comment]
this occurred again today in the field.  Cleared but is an issue with the users...Ingrid, writing on Monjur's account. it is a CHCP user

**Comment by AkubaD** [comment]
Hi @RahmanMonjur 
Have you seen this issue again? Ingrid said she saw it in the field during her last visit for the audit. Did you see it at any point during this training? Could you please ask the facility workers you are training if they have experienced this? We have not seen it on Dev for months now. I would like to confirm if it will be fixed with the Dev updates, or if this is perhaps a network issue that is mostly observed in the field.
Looking forward to your feedback

**Comment by RahmanMonjur** [comment]
@AkubaD Yes, we had the problem in the training. one new thing also occurred this time, Org Unit was not pre-filled while registering a client.

**Comment by AkubaD** [comment]
@RahmanMonjur Since you trained on a copy of Dev, then this is still an issue. The problem though is how to replicate it...
Could you give more information on this new issue of org unit not prefilled when registering a client
What user was it

**Comment by lukasewvd** [comment]
I have spent some time trying to reproduce this issue without any success. For next time this happens try to log details about: which orgunit/program, how was the internet connection (slow/fast), what fields where filled in when searching, etc. Basically try to log as much detailed information as possible as this might help us figure out the casue of the bug.

**Comment by AkubaD** [comment]
Thanks for following up @lukasewvd . I have not been able to replicate it either.
@RahmanMonjur could you please send an update with the info Lukas requested when/if it shows up again?
